appveyor.yml

```diff
@@ -60,11 +59,11 @@ install:
 
   # Install the dependencies of the project.
   - ps: Add-AppveyorMessage "Installing conda packages..."
-  - "conda install -yq -c conda-forge %TEST_DEPS%"
   - "conda install -yq -c conda-forge %DEPS%"
```

Comment:
For win32, this should pull matploltib=2.2.3 from the default channel since this is not in the conda-forge channel, but it doesn't work...